Seq.App.HipChat/HipChatReactor.cs

```diff
@@ -96,5 +103,70 @@ public class HipChatReactor : Reactor, ISubscribeTo<LogEventData>
                 }
             }
         }
+
+        private void AddMessage(Event<LogEventData> evt, StringBuilder msg)
+        {
+            var messageTemplateToUse = MessageTemplate;
+
+            if (string.IsNullOrWhiteSpace(MessageTemplate))
+            {
+                MessageTemplate = "<strong>[Level]:</strong> [RenderedMessage]";
+            }
+
+            msg.AppendFormat(SubstitutePlaceholders(messageTemplateToUse, evt));
+        }
+
+        private string SubstitutePlaceholders(string messageTemplateToUse, Event<LogEventData> evt)
+        {
+            var data = evt.Data;
+            var eventType = evt.EventType;
+
+            var placeholders = data.Properties.ToDictionary(k => k.Key.ToLower(), v => v.Value);
+
+            AddValueIfKeyDoesntExist(placeholders, "EventType", eventType);
+            AddValueIfKeyDoesntExist(placeholders, "RenderedMessage", data.RenderedMessage);
```

Comment:
Good spot, I've added it. 

Will give it a while in the way you've suggested cheers